python/ray/util/sgd/torch/examples/image_models/train.py

```diff
@@ -0,0 +1,150 @@
+# Based on work by Ross Wightman as part of the timm package
+# (see LICENSE_THIRDPARTY)
+#
+# As modified by
+# - Maksim Smolin in 2020
+
+# Note: other authors MUST include themselves in the above copyright notice
+#       in order to abide by the terms of the Apache license
+
+from os.path import join
+
+from tqdm import trange
+
+import torch.nn as nn
+
+from timm.data import Dataset, create_loader
+from timm.data import resolve_data_config, FastCollateMixup
+from timm.models import create_model, convert_splitbn_model
+from timm.optim import create_optimizer
+from timm.utils import setup_default_logging
+
+import ray
+from ray.util.sgd.utils import BATCH_SIZE
+
+from ray.util.sgd import TorchTrainer
+# from ray.util.sgd.torch import TrainingOperator
+
+from ray.util.sgd.torch.examples.image_models.args import parse_args
+
+
+def model_creator(config):
+    args = config["args"]
+
+    model = create_model(
+        "resnet101",  # args.model,
+        pretrained=args.pretrained,
+        num_classes=args.num_classes,
+        drop_rate=args.drop,
+        drop_connect_rate=args.drop_connect,  # DEPRECATED, use drop_path
+        drop_path_rate=args.drop_path,
+        drop_block_rate=args.drop_block,
+        global_pool=args.gp,
+        bn_tf=args.bn_tf,
+        bn_momentum=args.bn_momentum,
+        bn_eps=args.bn_eps,
+        checkpoint_path=args.initial_checkpoint)
+
+    if args.split_bn:
+        assert args.num_aug_splits > 1 or args.resplit
+        model = convert_splitbn_model(model, max(args.num_aug_splits, 2))
```

Comment:
were you going to remove this?

Reply:
I just disabled the arguments for this, it's always False rn